Microsoft.Web.Graph.WebRole/Controllers/DocPageController.cs

```diff
@@ -5,20 +5,24 @@
 using System.Web.Mvc;
 using Microsoft.Web.Graph.WebRole.Util;
 using DynamicDocs.Models;
-using Microsoft.Web.Portal.Common.Services;
+using Microsoft.Web.Portal.Common.Culture;
+using Microsoft.Web.Portal.Common.Logging;
 
 namespace Microsoft.Web.Graph.WebRole.Controllers
 {
     public class DocPageController : BaseController
     {
         private ICultureService _cultureService = null;
+        private ILogger _logger = null;
 
-        public DocPageController(ICultureService cultureService)
+        public DocPageController(ICultureService cultureService, ILogger logger)
         {
             _cultureService = cultureService;
+            _logger = logger;
         }
         public ActionResult GetDocPage(string culture, string docPath)
         {
+            _logger.Log(LogLevel.Debug, "GetDocPage");
```

Comment:
yes, but this was Jinlin's prototype code, so I will let Jinlin or who ever takes the code, should do that, the purpose here is to verify you can log some text in your controller